### PR TITLE
refactor(probabilistic_occupancy_grid_map): rework parameters

### PR DESCRIPTION
**Pull request by tby-udel**
## Description
Implement the ROS Node configuration layout described in https://github.com/orgs/autowarefoundation/discussions/3371 for the `probabilistic_occupancy_grid_map` package.

 * Renamed the grid_map_param.yaml file to grid_map.param.yaml file.
 * Move the default parameter file to a standardized location. Created a lot of JSON files for schema.
 * Updated readme file.

## Tests performed
Package built and launched locally.

## Notes for reviewers
 Nothing inside the src code part was changed, only taking the parameters out.

## Interface changes
Parameter values are now required to be passed to the node when launched.

## Effects on system behavior
None

## Pre-review checklist for the PR author
The PR author **must** check the checkboxes below when creating the PR.

 * [x]  I've confirmed the [contribution guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/).
 * [x]  The PR follows the [pull request guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/).

 ## In-review checklist for the PR reviewers
 The PR reviewers **must** check the checkboxes below before approval.

 * [ ]  The PR follows the [pull request guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/).
 * [ ]  The PR has been properly tested.
 * [ ]  The PR has been reviewed by the code owners.
 
 ## Post-review checklist for the PR author
 The PR author **must** check the checkboxes below before merging.
 
 * [ ]  There are no open discussions or they are tracked via tickets.
 * [ ]  The PR is ready for merge.
 
 After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

**Review by yukkysaito — APPROVED** [review]
LGTM

**Comment by mitsudome-r** [comment]
@tby-udel Thanks for the PR. 

However, the PR seems to be failing in the CI for DCO check. When you push commits, could you push with signed-off commits?

For example by running the following command:
`git rebase -i --signoff origin/main`

**Comment by tby-udel** [comment]
Thank you for the reminder, I have pushed the signed off commit.

File: perception/probabilistic_occupancy_grid_map/schema/synchronized_grid_map_fusion_node.schema.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Synchronized Grid Map Fusion Node",
+  "type": "object",
+  "properties": {
+    "ros__parameters": {
```
Comment (mitsudome-r) [inline perception/probabilistic_occupancy_grid_map/schema/synchronized_grid_map_fusion_node.schema.json:6]: The CI is complaining about the syntax in this file.

I think the properties should start with `/**` as required field, but it is now starting with `ros__parameters`

**Comment by tby-udel** [comment]
The required changes have been made.

**Pull request merged**